### DISCUSSION TITLE
Welcome to e-mobility-charging-stations-simulator Discussions!

### DISCUSSION THREAD

**Discussion by jerome-benoit** [SAP/e-mobility-charging-stations-simulator#1232] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by singh786chandan** [comment]
Unable to start with OCPP 2.0.1. It's always taking 1.6 protocol after changing the version of protocol. in station-templates==>
e-mobility-charging-stations-simulator-main\src\assets\station-templates\
All templates are compatible with OCPP 1.6

**Comment by singh786chandan** [comment]
Getting below errors after changing version from 1.6 to 2.0.1

error: CS-SIEMENS: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration {"event":"addWorkerElement","name":"C","stack":"C: CS-SIEMENS: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration\n    at chargingStation (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\Helpers.ts:249:15)\n    at n.initialize (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:1794:5)\n    at new n (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:293:10)\n    at MessagePort.message (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStationWorker.ts:49:39)\n    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:843:20)\n    at MessagePort.<anonymous> (node:internal/per_context/messageport:23:28)"}
error: CS-KEBA-00001: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration {"event":"addWorkerElement","name":"C","stack":"C: CS-KEBA-00001: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration\n    at chargingStation (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\Helpers.ts:249:15)\n    at n.initialize (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:1794:5)\n    at new n (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:293:10)\n    at MessagePort.message (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStationWorker.ts:49:39)\n    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:843:20)\n    at MessagePort.<anonymous> (node:internal/per_context/messageport:23:28)"}
error: CS-KEBA-00002: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration {"event":"addWorkerElement","name":"C","stack":"C: CS-KEBA-00002: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration\n    at chargingStation (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\Helpers.ts:249:15)\n    at n.initialize (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:1794:5)\n    at new n (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:293:10)\n    at MessagePort.message (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStationWorker.ts:49:39)\n    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:843:20)\n    at MessagePort.<anonymous> (node:internal/per_context/messageport:23:28)"}
error: CS-ABB-00001: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration {"event":"addWorkerElement","name":"C","stack":"C: CS-ABB-00001: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration\n    at chargingStation (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\Helpers.ts:249:15)\n    at n.initialize (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:1794:5)\n    at new n (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:293:10)\n    at MessagePort.message (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStationWorker.ts:49:39)\n    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:843:20)\n    at MessagePort.<anonymous> (node:internal/per_context/messageport:23:28)"}
error: CS-ABB-00002: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration {"event":"addWorkerElement","name":"C","stack":"C: CS-ABB-00002: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration\n    at chargingStation (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\Helpers.ts:249:15)\n    at n.initialize (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:1794:5)\n    at new n (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:293:10)\n    at MessagePort.message (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStationWorker.ts:49:39)\n    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:843:20)\n    at MessagePort.<anonymous> (node:internal/per_context/messageport:23:28)"}
error: CS-EVLINK-00001: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration {"event":"addWorkerElement","name":"C","stack":"C: CS-EVLINK-00001: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration\n    at chargingStation (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\Helpers.ts:249:15)\n    at n.initialize (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:1794:5)\n    at new n (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:293:10)\n    at MessagePort.message (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStationWorker.ts:49:39)\n    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:843:20)\n    at MessagePort.<anonymous> (node:internal/per_context/messageport:23:28)"}
error: CS-EVLINK-00002: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration {"event":"addWorkerElement","name":"C","stack":"C: CS-EVLINK-00002: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration\n    at chargingStation (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\Helpers.ts:249:15)\n    at n.initialize (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:1794:5)\n    at new n (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:293:10)\n    at MessagePort.message (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStationWorker.ts:49:39)\n    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:843:20)\n    at MessagePort.<anonymous> (node:internal/per_context/messageport:23:28)"}
error: CS-EVLINK-00003: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration {"event":"addWorkerElement","name":"C","stack":"C: CS-EVLINK-00003: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration\n    at chargingStation (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\Helpers.ts:249:15)\n    at n.initialize (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:1794:5)\n    at new n (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:293:10)\n    at MessagePort.message (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStationWorker.ts:49:39)\n    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:843:20)\n    at MessagePort.<anonymous> (node:internal/per_context/messageport:23:28)"}
error: CS-EVLINK-00004: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration {"event":"addWorkerElement","name":"C","stack":"C: CS-EVLINK-00004: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration\n    at chargingStation (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\Helpers.ts:249:15)\n    at n.initialize (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:1794:5)\n    at new n (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:293:10)\n    at MessagePort.message (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStationWorker.ts:49:39)\n    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:843:20)\n    at MessagePort.<anonymous> (node:internal/per_context/messageport:23:28)"}
error: CS-SCHNEIDER-00001: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration {"event":"addWorkerElement","name":"C","stack":"C: CS-SCHNEIDER-00001: OCPP 2.0.1 requires at least one EVSE defined in the charging station template/configuration\n    at chargingStation (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\Helpers.ts:249:15)\n    at n.initialize (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:1794:5)\n    at new n (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStation.ts:293:10)\n    at MessagePort.message (C:\\Chandan\\e-mobility-charging-stations-simulator-main\\e-mobility-charging-stations-simulator-main\\src\\charging-station\\ChargingStationWorker.ts:49:39)\n    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:843:20)\n    at MessagePort.<anonymous> (node:internal/per_context/messageport:23:28)"}
